ubuild.py

```diff
@@ -0,0 +1,11 @@
+from uranium import current_build
+
+current_build.config.set_defaults({
+    "package_name": "intake-nested-yaml-catalog",
+    "module": "intake_nested_yaml_catalog",
+    "binaries": {"enabled": True},
+})
+
+current_build.packages.install(".", develop=True)
```

Comment:
this should be unneeded. orbital-core will install the develop package as part of the main task: https://github.com/zillow/orbital-core/blob/master/orbital_core/build/tasks.py#L6